API/PROBLEMS/with_identifiers/strange

```diff
@@ -0,0 +1,11 @@
+--------------------------------------------------------------------------------
+
+INSTANCES
+
+- Namegen.next_ident_away
+- Term.types (* plural: misleading *)
+- EConstr.types (* plural: misleading *)
```

Comment:
That's just a typical thing when we want an identifier that is already a keyword. Some prefer `types_`. But in any case, I don't feel like this is a big issue.